pkg/features/kube_features.go

```diff
@@ -909,7 +910,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	PodSecurity:                                    {Default: true, PreRelease: featuregate.Beta},
 	ReadWriteOncePod:                               {Default: false, PreRelease: featuregate.Alpha},
 	CSRDuration:                                    {Default: true, PreRelease: featuregate.Beta},
-	DelegateFSGroupToCSIDriver:                     {Default: false, PreRelease: featuregate.Alpha},
+	DelegateFSGroupToCSIDriver:                     {Default: true, PreRelease: featuregate.Beta},
```

Comment:
Do we have feature-tagged e2e tests that need to be updated?

Reply:
Not for this feature. Besides the test change in fsgroupchangepolicy.go above I'm planning to add tests explicitly verifying fsgroup delegation in this release.